output/cloud/expv2/mapping.go

```diff
@@ -52,44 +52,39 @@ func addBucketToTimeSeriesProto(
 	timeSeries *pbcloud.TimeSeries,
 	mt metrics.MetricType,
 	time time.Time,
-	sink metrics.Sink,
+	sink metricValue,
 ) {
 	if timeSeries.Samples == nil {
 		initTimeSeriesSamples(timeSeries, mt)
 	}
 
-	switch mt {
-	case metrics.Counter:
-		c := sink.(*metrics.CounterSink).Value //nolint: forcetypeassert
+	switch typedSink := sink.(type) {
```

Comment:
the type is now `metricValue` instead of `Sink` so a rename of both is likely in order.